pkg/webui/account/views/app/index.js

```diff
@@ -0,0 +1,100 @@
+// Copyright © 2020 The Things Network Foundation, The Things Industries B.V.
+//
+// Licensed under the Apache License, Version 2.0 (the "License");
+// you may not use this file except in compliance with the License.
+// You may obtain a copy of the License at
+//
+//     http://www.apache.org/licenses/LICENSE-2.0
+//
+// Unless required by applicable law or agreed to in writing, software
+// distributed under the License is distributed on an "AS IS" BASIS,
+// WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+// See the License for the specific language governing permissions and
+// limitations under the License.
+
+import { hot } from 'react-hot-loader/root'
+import { useSelector, useDispatch } from 'react-redux'
+import React, { useEffect } from 'react'
+import { Switch, Route } from 'react-router-dom'
+import { ConnectedRouter } from 'connected-react-router'
+import { Helmet } from 'react-helmet'
+
+import { ToastContainer } from '@ttn-lw/components/toast'
+
+import ErrorView from '@ttn-lw/lib/components/error-view'
+import { FullViewError } from '@ttn-lw/lib/components/full-view-error/error'
+
+import Landing from '@account/views/landing'
+import Authorize from '@account/views/authorize'
+
+import PropTypes from '@ttn-lw/lib/prop-types'
+import dev from '@ttn-lw/lib/dev'
+import {
+  selectApplicationSiteName,
+  selectApplicationSiteTitle,
+  selectPageData,
+} from '@ttn-lw/lib/selectors/env'
+import { setStatusOnline } from '@ttn-lw/lib/store/actions/status'
+
+import { selectUser } from '@account/store/selectors/user'
+
+import Front from '../front'
+
+const siteName = selectApplicationSiteName()
+const siteTitle = selectApplicationSiteTitle()
+const pageData = selectPageData()
+
+const AccountApp = ({ history }) => {
+  const user = useSelector(selectUser)
+  const dispatch = useDispatch()
+
+  useEffect(() => {
+    const handleConnectionStatusChange = ({ type }) => {
+      dispatch(setStatusOnline(type === 'online'))
+    }
+
+    window.addEventListener('online', handleConnectionStatusChange)
+    window.addEventListener('offline', handleConnectionStatusChange)
+
+    return () => {
+      window.removeEventListener('online', handleConnectionStatusChange)
+      window.removeEventListener('offline', handleConnectionStatusChange)
+    }
+  }, [dispatch])
+
+  if (pageData && pageData.error) {
+    return (
+      <ConnectedRouter history={history}>
+        <FullViewError error={pageData.error} />
```

Comment:
`<FullViewError />` also requires the `onlineStatus` prop. Otherwise the offline status is shown.